### PR TITLE
docs: add ngVikings 2019 to events page

### PR DESCRIPTION
**Pull request by softchris**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
ngVikings2019 entry missing in events.html

## What is the new behavior?
ngVikings2019 entry added to events.html

## Does this PR introduce a breaking change?

- [ ] Yes
- [ x] No

### CODE REVIEW

**Review by StephenFluin — CHANGES_REQUESTED** [review]
Can you make sure the commit message matches our contributing guidelines?

I'm also seeing a lot of unrelated whitespace changes. Can you make sure you only touch the new event, and that it matches the existing formatting?

**Comment by softchris** [comment]
> Can you make sure the commit message matches our contributing guidelines?
> 
> I'm also seeing a lot of unrelated whitespace changes. Can you make sure you only touch the new event, and that it matches the existing formatting?

changed the title, hope that's the format you mean. Also fixed the formatting I think..

File: aio/content/marketing/events.html [hunk 1]
```diff
@@ -25,6 +25,12 @@ <h1 class="banner-headline no-toc no-anchor">Events</h1>
       <td>Gurgaon, India</td>
       <td>February 23, 2019</td>
     </tr>
+    <!-- ngVikings 2019-->
+    <tr>
+      <th><a href="https://ngvikings.org/" title="ngVikings">ngVikings</a></th>
+      <td>Copenhagen, Denmark</td>
+      <td>May 26-28, 2019</td>
```
Comment (trotyl) [inline aio/content/marketing/events.html]: In the website it's said to be **May 27-28th, 2019**, better to align them?

Comment (softchris) [reply aio/content/marketing/events.html]: well 26th is workshop day, 27-28 is conference.. Yes we will update the web page to say that :)

**Comment by StephenFluin** [comment]
Now I think you'll need to squash the commits. 

https://twitter.com/stephenfluin/status/1009904095073718275

**Comment by softchris** [comment]
> Now I think you'll need to squash the commits.
> 
> https://twitter.com/stephenfluin/status/1009904095073718275

ok I squashed it.. Happy now? :)

**Comment by brandonroberts** [comment]
The git commit message needs to be updated to `docs: add ngVikings 2019 to events page`

**Comment by webmaxru** [comment]
@softchris any status on changing commit message?

**Review by brandonroberts — APPROVED** [review]
LGTM. @StephenFluin will give final approval.

**Review by StephenFluin — APPROVED** [review]

**Pull request closed without merging**